cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsInstanceConnector.java

```diff
@@ -134,18 +134,21 @@ public String getConsoleOutput(AuthenticatedContext authenticatedContext, CloudI
         AmazonEC2Client amazonEC2Client = awsClient.createAccess(new AwsCredentialView(ac.getCloudCredential()),
                 ac.getCloudContext().getLocation().getRegion().value());
         List<CloudInstance> affectedVms = new ArrayList<>();
+        List<CloudVmInstanceStatus> rebootedVmsStatus = new ArrayList<>();
+
         try {
             if (!vms.isEmpty()) {
                 List<CloudVmInstanceStatus> statuses = check(ac, vms);
-                doReboot(affectedVms, amazonEC2Client, getStarted(statuses));
-                doStart(affectedVms, ac, getStopped(statuses));
+                doStop(new ArrayList<>(), ac, getStarted(statuses));
+                statuses = check(ac, vms);
+                doStart(affectedVms, ac, getStopped(statuses), rebootedVmsStatus);
```

Comment:
instead of passing `affectedVms` list to `doStop` and `doStart` it would be nicer if they return with a list and we can use that

Reply:
I miss some action here. what happens here is:
- issue stop for VMs which are running
- collect the VM statuses
- start VMs which are stopped

In theory all VMs should be stopped at the point we are collecting the statuses after stop. I think we should at least match that the requested VMs for reboot are all stopped and have a log here if any of them is not stopped

Reply:
Yes. I returned _rebootedVmsStatus_ from doStart() method. Thanks for the suggestion.

I also added a specific log check to see if any requested VMs for reboot were not stopped  
-_logInvalidStatuses(getNotStopped(statuses));_